ajax/libs/jquery.ime/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "jquery.ime",
+  "version": "0.2.0",
+  "description": "jQuery based internationalization library",
+  "keywords": [
+    "internationalization",
+    "localization",
+    "ime",
+    "jquery",
+    "l10n"
```

Comment:
Let's add more keywords(like `intl`) to help people to search this lib.
![screen shot 2018-04-27 at 11 02 06 am](https://user-images.githubusercontent.com/8349587/39342462-bc3c3d5e-4a0a-11e8-8896-201aa112d9ac.png)
Thanks